styleguide/derek/view-templates/stories/_stories.scss

```diff
@@ -216,33 +254,73 @@ $default-bg-img: 'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.
 }
 
 .storyOverview-itemInner {
-  border: 1px solid $gray-light;
-  padding: 5px;
+  border: 1px solid fade-out($white, .7);
 }
 
 .storyOverview-itemLink {
   @include flex-box;
   @include flex-center;
-  background: $gray-base;
   height: 125px;
+  overflow: hidden;
+  position: relative;
   text-align: center;
   text-decoration: none;
 
   &:hover {
-    background-color: $gray-dark;
+    background-color: fade-out($white, .9);
     text-decoration: none;
+    transition-duration: .2s;
   }
 }
 
+@include storyGridImg('.storyOverview-itemLink-withImg');
+
 .storyOverview-logo {
   max-height: 50%;
   max-width: 50%;
+  z-index: 1;
 }
 
 .storyOverview-logoText {
   color: $white;
   font-size: 1.8rem;
   max-width: 80%;
+  z-index: 1;
+}
+
+.rsFilter-storiesBarTitle {
+  color: fade-out($white, .3);
+}
+
+.rsFilter-storiesContent {
+  background-color: $content-gray;
+}
+
+.rsFilter-storiesBar {
+  background-color: $filterbar-gray;
+}
+
+.rsFilter-storiesPager {
+
+  & > .pager-previous {
+    color: $stories-link;
+  }
```

Comment:
You have `.rsFilter-storiesPager-prev` just a few lines below this. Is this still necessary? Same with previous below.